src/js/jsx/shared/ColorPicker.jsx

```diff
@@ -403,18 +406,121 @@ define(function (require, exports, module) {
-                    style={{
-                        left: this._getPercentageValue(this.props.x),
-                        bottom: this._getPercentageValue(this.props.y)
-                    }} />
-              </div>
+                    className="color-picker__colortype">
+                    <div
+                        className="color-picker__colortype__thumb empty">
+                    </div>
+                    <Gutter/>
+                    <div
+                        className="color-picker__colortype__thumb selected">
+                    </div>
+                    <Gutter/>
+                    <Gutter/>
+                    <NumberInput
+                            ref="left"
+                            size="column-11"
+                            placeholder="rgba(255,255,255,1)"
+                            value="rgba(255,255,255,1)"/>
+                  </div>
+            );
+        }
+    });
+    /**
+     * Placeholder color RGB and HSB input values.
+     * 
+     * @constructor
+     */
+    var ColorFields = React.createClass({
+        render: function () {
+            return (
+                <div className="color-picker__rgbhsb">
+                    <div className="formline">
+                        <Label>
+                            R
+                        </Label>
+                        <Gutter/>
+                        <NumberInput
+                            ref="left"
+                            size="column-5"
+                            placeholder="255" />
+                    </div>
+                    <div className="formline">
+                        <Label>
+                            G
+                        </Label>
+                        <Gutter/>
+                        <NumberInput
+                            ref="left"
+                            size="column-5"
+                            placeholder="255" />
+                    </div>
+                    <div className="formline">
+                        <Label>
+                            B
+                        </Label>
+                        <Gutter/>
+                        <NumberInput
+                            ref="left"
+                            size="column-5"
+                            placeholder="255" />
+                    </div>
+                    <Gutter/>
+                    <Gutter/>
+                    <div className="formline">
+                        <Label>
+                            H
+                        </Label>
+                        <Gutter/>
+                        <NumberInput
+                            ref="left"
+                            size="column-5"
+                            placeholder="255" />
+                    </div>
+                    <div className="formline">
+                        <Label>
+                            S
+                        </Label>
+                        <Gutter/>
```

Comment:
If possible, I'd rather not use these Gutter elements and create space with CSS instead